### PR TITLE
[Doc] Add doc on how to configure max subscriptions per topic at the topic level 

### PR DESCRIPTION
**Pull request by Jennifer88huang**
Master Issue: #8866 
Related PR:  #8948  

### Motivation
In  #8948, we support configuring max subscriptions at topic level.

### Modifications

1. Add related doc content for this feature.
2. Fix some minor typos.

### CODE REVIEW

**Comment by Jennifer88huang** [comment]
@315157973 we might need to add related options for `set-max-subscriptions` if we have. Could you provide available options? Thank you.

File: site2/docs/reference-pulsar-admin.md [hunk 1]
```diff
@@ -2383,6 +2386,30 @@ Usage
 $ pulsar-admin topics remove-deduplication tenant/namespace/topic
 ```
 
+### `set-max-subscriptions`
+Set the maximum number of subscriptions for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-max-subscriptions tenant/namespace/topic options
```
Comment (315157973) [inline site2/docs/reference-pulsar-admin.md:2394]: It looks good, but I have some questions:
What does `tenant/namespace/topic` mean?

If we use this command,a topic name should be passed in here, such as:
persistent://tenant/namespace/topic

The entire command looks like this:
pulsar-admin topics set-max-subscriptions [topic name] [options]

**Comment by 315157973** [comment]
Now this command is also called `get-max-subscriptions-per-topic` at topic-level. But I think it would be more reasonable to call `get-max-subscriptions`. I will post a PR to fix this problem later.
Because, we have `get-max-producers-per-topic` at namespace-level, but it is called `get-max-producers` at topic-level. We should be consistent with these rules.

**Comment by 315157973** [comment]
Here is the PR https://github.com/apache/pulsar/pull/9750

File: site2/docs/reference-pulsar-admin.md [hunk 2]
```diff
@@ -1852,6 +1852,9 @@ Subcommands
 * `get-inactive-topic-policies`
 * `set-inactive-topic-policies`
 * `remove-inactive-topic-policies`
+* `set-max-subscriptions`
+* `get-max-subscriptions`
+* `remove-max-subscriptions`
```
Comment (Anonymitaet) [inline site2/docs/reference-pulsar-admin.md:1857]: Need to add them to [pulsar-admin website](http://pulsar.apache.org/tools/pulsar-admin/2.8.0-SNAPSHOT/#-em-set-max-subscriptions-per-topic-em-)? or is `set-max-subscriptions` same to `set-max-subscriptions-per-topic`? Same comments for the other two commands.

Comment (Jennifer88huang) [reply site2/docs/reference-pulsar-admin.md:1857]: `set-max-subscriptions` is the same to `set-max-subscriptions-per-topic`, fixed in PR #9750

**Comment by Jennifer88huang** [comment]
@315157973 @Anonymitaet updated, PTAL again, thank you.

**Review by Anonymitaet — APPROVED** [review]

**Comment by Jennifer88huang** [comment]
@315157973  @codelipenghui if there is no further issue, could you approve and merge it? If you have any concern, feel free to comment. Thank you.

**Review by codelipenghui — APPROVED** [review]

**Review by 315157973 — APPROVED** [review]

**Pull request merged**